_articles/linux-terminal_for-beginners.md

```diff
@@ -26,25 +26,38 @@ A box like the one below will appear.
 
 When you run a command with **sudo** in front of it, You'll be prompted for your password, but when you type it, it won't show the letters as you type in the terminal, but type your password anyway and press <kbd>enter</kbd>.
 
-#### Frequently Used Commands
+#### Frequently Used Commands:
+*Omit the brackets when entering the command.*
+
+This series of commands will update the sources.list (a list telling your Ubuntu Operating System where to look for software and updates.)
+
+`sudo apt update`
+
+When using the 'upgrade' option of the command 'apt' your System will download and apply any updates to your System76.
+
+`sudo apt upgrade`
+
+The "man" command is short for manual. You can use it like so: 
+
+`man sources.list`
 
 ### Add an Application
 
-`sudo apt-get install [insert application name here]`
+`sudo apt install [insert application name here]`
 
 *Omit the brackets when entering the command.*
 
 ## Example:
 
-`sudo apt-get install google-chrome-stable`
+`sudo apt install google-chrome-stable`
 
 ### Remove a package or application
 
-`sudo apt-get purge google-chrome-stable`
+`sudo apt purge google-chrome-stable`
 
-`sudo apt-get autoremove google-chrome-stable`
+`sudo apt autoremove google-chrome-stable`
```

Comment:
`apt autoremove` doesn't take any more modifiers.  It will clean up every unneeded package as a bare command.  Remove the google-chrome-stable here because it is misleading.